FoodBookApp/DAO/Firebase/SpotDAOFirebase.swift

```diff
@@ -41,5 +41,10 @@ class SpotDAOFirebase: SpotDAO {
         return try snapshot.data(as: Review.self)
     }
     
+    func updateSpot(documentId: String, reviewId: String) async throws {
+        let spotRef = collection.document(documentId)
+        let reviewRef = client.db.collection("reviews").document(reviewId)
+        try await spotRef.updateData(["reviewData.userReviews": FieldValue.arrayUnion([reviewRef])])
+    }
```

Comment:
Notice this function receives reviewId, when an update wouldn't necesarilly include it. However, I did it this way since the only time we're updating a spot from the app is to add the reference to the review